src/scale.jl

```diff
@@ -313,7 +325,7 @@ anything in the data that's not respresented in `levels` will be set to
 default order.
 
 See also [`group_discrete`](@ref), [`shape_discrete`](@ref), 
-[`size_discrete`](@ref), and [`linestyle_discrete`](@ref).
+[`size_discrete`](@ref), and [`linestyle_discrete`](@ref), and [`alpha_discrete`](@ref).
```

Comment:
the "and" before linestyle_discrete could be removed as it was in #1252

Reply:
Done